libraries/AP_Scripting/examples/BQ40Z_bms_shutdown.lua

```diff
@@ -0,0 +1,74 @@
+-- TI BQ40Z BMS shutdown script
+---@diagnostic disable: need-check-nil
```

Comment:
This is cheating. if you post the warning I'm sure we can work out how fix it.

Reply:
I removed this script as it was for the table-based transfer function. I can adapt it but as it duplicates the functionality in #27288 I just removed it.